openedx/core/djangoapps/plugins/README.rst

```diff
@@ -1,248 +0,0 @@
-Django App Plugins
```

Comment:
This app is greatly simplified, but still exists, right? We should have a README that describes the purpose of the app.